packages/sw-runtime-caching/src/lib/constants.js

```diff
@@ -35,5 +35,4 @@ export const behaviorCallbacks = [
   'cacheDidUpdate',
   'cacheWillUpdate',
   'fetchDidFail',
-  'requestWillFetch',
```

Comment:
Though this might be added in at some point, `RequestWrapper` doesn't currently support `requestWillFetch`, so I'm removing it from here and from the JSDoc.